### PR TITLE
Documentation 3.0/cookbook integration new build script

### PR DESCRIPTION
**Pull request by Simran-B**
Had to replace `_` with `&lowbar;` twice to fix incorrect translation to `<em>` - which only seems to occur on my system (Windows/Cygwin).

    [Link label that breaks: GRAPH_* functions](http://example.org)

### CODE REVIEW

**Comment by dothebart** [comment]
@janavolkova9 ^^^ Hubspot URLS

File: Documentation/Books/Cookbook/Graph/JavaDriverGraphExampleVertex.mdpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# How to use an example vertex with the java driver
+
+## Problem
+
+**Note**: Arango 2.2 and the corresponding javaDriver is needed.
```
Comment (dothebart) [inline Documentation/Books/Cookbook/Graph/JavaDriverGraphExampleVertex.mdpp]: lol.

**Comment by dothebart** [comment]
LGTM.

**Comment by Simran-B** [comment]
With MarkdownPP disabled, it actually breaks because the markup uses MarkdownPP style headlines. I got a Python script to fix that, but I wonder if I should remove MarkdownPP entirely and rename *.mdpp to *.md like we did for 3.2 (and do the same in 2.8 and 3.1 branches).

**Comment by dothebart** [comment]
if its just running a script, please do so. Remember, the same is also required for 2.8

**Comment by Simran-B** [comment]
- Converting headlines is a matter of running an existing script
- Removing MarkdownPP entirely should be a matter of cherry-picking
- Minor differences in build.sh because of .mdpp file extensions can be removed by applying a 3.2 branch diff for that file
- Renaming all .mdpp files to .md can be done with a shell command
- A change to all book.json files will be necessary regarding the edit-link plugin, which corrects the Github edit URLs to .mdpp extensions

Am I overlooking something?

Not sure if it's worth the time to fix or remove outdated cookbook recipes in the legacy branches.

**Comment by dothebart** [comment]
Go ahead. 

for i in $(find -name *mdpp); do git mv $i $(echo $i |sed "s;mdpp$;md;"); done 

send me a note once you're done so I can test it.

**Pull request merged**